Objects/typevarobject.c

```diff
@@ -1,4 +1,4 @@
-// TypeVar, TypeVarTuple, and ParamSpec
+// TypeVar, TypeVarTuple, TypeAlias and ParamSpec
```

Comment:
```suggestion
// TypeVar, TypeVarTuple, ParamSpec, and TypeAlias
```
Keep similar things together